comps/comps-foreman-plugins-rhel7.xml

```diff
@@ -165,10 +167,8 @@
       <packagereq type="default">rubygem-smart_proxy_abrt-doc</packagereq>
       <packagereq type="default">rubygem-smart_proxy_chef-doc</packagereq>
       <packagereq type="default">rubygem-smart_proxy_discovery-doc</packagereq>
-      <packagereq type="default">rubygem-smart_proxy_dynflow-doc</packagereq>
```

Comment:
Oh, probably because they've been cherry picked back.  Sorry, please leave them in!